docs/class_diagram/class_diagram.md

```diff
@@ -0,0 +1,65 @@
+```mermaid
+classDiagram
+    Game --> "*" Player : has
+    Game --> "1" Deck : has
+    Game --> "5" DiscardPile : has
+    Player --> "1" Hand : has
+    Hand --> "*" Card : contains
+    Player --> "*" Expedition : has
+    Expedition --> "*" Card : contains
+    AbstractPile <|-- Deck
+    AbstractPile <|-- DiscardPile
+    AbstractPile <|-- Hand
+    class Game{
+      #players
+      #deck
+      #discardPiles
+      #currentPlayer
+      +start()
+      +end()
+      +calculateScores()
+      +turn(Player, dict)
+    }
+    class Player{
+      #name
+      #hand
+      #expeditions
```

Comment:
A pontszámot is el kell tárolni, mert a korábbi fordulók pontjait meg kell jegyezni.